### PR TITLE
docs: zh-cn translations for fail2ban setup

### PR DESCRIPTION
**Pull request by ghost**
share my Chinese translations for **usage >[ fail2ban setup](https://docs.gitea.io/en-us/fail2ban-setup/)**.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

File: docs/content/doc/usage/fail2ban-setup.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    identifier: "fail2ban-setup"
+---
+
+# 使用 Fail2ban 阻止攻击者的暴力登录
+
+**Fail2ban 检查客户端登录日志，将多次登录失败的客户端识别为攻击者并在一段时间内阻止其访问服务。如果你的实例是公开的，这一点尤其重要。请管理员仔细设置 fail2ban，错误的配置将导致防火墙阻止你访问自己的服务器。**
+
+Gitea 会在日志文件 `log/gitea.log` 中记录登录失败的 CLI、SSH 或 HTTP 客户端 IP 地址，而你需要将 Gitea 的日志输出模式从默认的 `console` 更改为 `file`。这表示将日志输出到文件，使得 fail2ban 可以定期扫描日志内容。
+
+
```
Comment (wxiaoguang) [inline docs/content/doc/usage/fail2ban-setup.zh-cn.md]:
```suggestion
```

File: docs/content/doc/usage/fail2ban-setup.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+额外注意，在 Gitea 的配置文件 `app.ini` 中存在下列默认值：
+
+```
+REVERSE_PROXY_LIMIT = 1
+REVERSE_PROXY_TRUSTED_PROXIES = 127.0.0.0/8,::1/128
+```
+
+`REVERSE_PROXY_LIMIT` 限制反向代理服务器的层数，设置为 `0` 表示不使用这些标头。`REVERSE_PROXY_TRUSTED_PROXIES` 表示受信任的反向代理服务器网络地址，经过该网络地址转发来的流量会经过解析 `X-Real-IP` 头部得到真实客户端地址。（参考 [configuration cheat sheet](https://docs.gitea.io/en-us/config-cheat-sheet/#security-security)）
```
Comment (wxiaoguang) [inline docs/content/doc/usage/fail2ban-setup.zh-cn.md]:
```suggestion
`REVERSE_PROXY_LIMIT` 限制反向代理服务器的层数，设置为 `0` 表示不使用这些标头。
`REVERSE_PROXY_TRUSTED_PROXIES` 表示受信任的反向代理服务器网络地址，
经过该网络地址转发来的流量会经过解析 `X-Real-IP` 头部得到真实客户端地址。
（参考 [configuration cheat sheet](https://docs.gitea.io/en-us/config-cheat-sheet/#security-security)）

```

File: docs/content/doc/usage/fail2ban-setup.zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+enabled = true
+filter = gitea
+logpath = /var/lib/gitea/log/gitea.log
+maxretry = 10
+findtime = 3600
+bantime = 900
+action = iptables-allports
+```
+
+如果你的 Gitea 实例运行在 Docker 容器中，并且直接将容器端口暴露到外部网络，你还需要添加 `chain="FORWARD"` 到监狱规则配置文件 `/etc/fail2ban/jail.d/gitea-docker.conf` 以适应 Docker 的网络转发规则。但如果你在容器的宿主机上使用 Nginx 反向代理连接到 Gitea 则无需这样配置。
```
Comment (wxiaoguang) [inline docs/content/doc/usage/fail2ban-setup.zh-cn.md]:
```suggestion
如果你的 Gitea 实例运行在 Docker 容器中，并且直接将容器端口暴露到外部网络，
你还需要添加 `chain="FORWARD"` 到监狱规则配置文件 `/etc/fail2ban/jail.d/gitea-docker.conf`
 以适应 Docker 的网络转发规则。但如果你在容器的宿主机上使用 Nginx 反向代理连接到 Gitea 则无需这样配置。
```

File: docs/content/doc/usage/fail2ban-setup.zh-cn.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+logpath = /var/lib/gitea/log/gitea.log
+maxretry = 10
+findtime = 3600
+bantime = 900
+action = iptables-allports
+```
+
+如果你的 Gitea 实例运行在 Docker 容器中，并且直接将容器端口暴露到外部网络，
+你还需要添加 `chain="FORWARD"` 到监狱规则配置文件 `/etc/fail2ban/jail.d/gitea-docker.conf`
+ 以适应 Docker 的网络转发规则。但如果你在容器的宿主机上使用 Nginx 反向代理连接到 Gitea 则无需这样配置。
```
Comment (wxiaoguang) [inline docs/content/doc/usage/fail2ban-setup.zh-cn.md]:
```suggestion
以适应 Docker 的网络转发规则。但如果你在容器的宿主机上使用 Nginx 反向代理连接到 Gitea 则无需这样配置。
```

**Comment by 6543** [comment]
.

**Pull request merged**